### PR TITLE
8

### PR DESCRIPTION
**Pull request by marioabz**
Documentation added to run.py with Sphinx format.
Waiting for approval.

### CODE REVIEW

**Comment by elohimf** [comment]
Remember to rebase before making a PR and after other PRs have been approved. There seems to be a lot of commits that have been merged already. Also, besides the milestone code, there should be a descriptive name (or description, repetitions aside) on the title of this PR.

**Review by atrevino89 — CHANGES_REQUESTED** [review]
Can you please send only your code, do a fetch & rebase then a forced push to your repo.

**Comment by marioabz** [comment]
Sorry guys, I was trying to merge into master. I already changed this PR to develop and now it's just 1 commit, the actual commit I'm trying to merge.

File: pyroute/run.py [hunk 1]
```diff
@@ -1,3 +1,23 @@
+""" Run module
+
+    The module 'Run' is the piece of code that is in charge of the
+
+    set up of the configuration and the tests of the scripts the user
+
+    wants to be tested.
+
```
Comment (eisaaclrubio) [inline pyroute/run.py:8]: @marioBZ7 can you remove the blank space between each line, from line 4 to line 7?

File: pyroute/run.py [hunk 2]
```diff
@@ -11,15 +31,38 @@
 
 
 class Run(object):
+    """
+    Class Run is intended to set the configuration requirements, ensure the init
 
+    of the Logger and the intitialization of a generic engine.
+   """
```
Comment (eisaaclrubio) [inline pyroute/run.py:38]: @marioBZ7 same here, please remove the line 36

File: pyroute/run.py [hunk 3]
```diff
@@ -11,15 +31,38 @@
...
+	""" execute_test method starts a counter at the logger and initialize
+
+	the generic engine.
+
+	The method **self.run_log** prints a default message to the logger.
+
+	The variable **session** is an instance of the generic engine and it
+
+	loads the configuration requirements.
+        """
```
Comment (eisaaclrubio) [inline pyroute/run.py:63]: @marioBZ7 same here with the lines 55 and 61.

**Review by eisaaclrubio — COMMENTED** [review]
@marioBZ7 When the documentation is generated the blank lines will force the comment to jump to the next line, please remove them so the documentation is written nicely.

**Comment by marioabz** [comment]
??????

**Pull request closed without merging**